### PR TITLE
add case study

### PR DESCRIPTION
**Pull request by jospoortvliet**
pls review @MariusBluem ;-)

### CODE REVIEW

File: page-tuberlin-whitepapersubmit.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    died($error_message);
+  }
+        function clean_string($string) {
+      $bad = array("content-type","bcc:","to:","cc:","href");
+      return str_replace($bad,"",$string);
+    }
+// the app review mailing list address
+    $email_from = "sales@nextcloud.com";
+    $email_jos = "jos@nextcloud.com";
+	$email_subject = "TU Berlin Casestudy download";
```
Comment (MariusBluem) [inline page-tuberlin-whitepapersubmit.php]: Case study

**Comment by jospoortvliet** [comment]
sry pushing as I have to get this live

**Comment by LukasReschke** [comment]
Sure because waiting a few minutes isn't doable and we'd rather again risk a broken website…

File: page-tuberlin-whitepapersubmit.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+				<h1><?php echo $l->t('Nextcloud in Education');?></h1>
+				<h2><?php echo $l->t('Easy, scaleable and well integrated');?></p>
+			</div>
+		</div>
+	</div>
+</div>
+<section class="section--whitepaper">
+    <div class="container text-center">
+        <h3>Thank you for your interest in our case study</h3>
+        <p>The case study has been sent to <?php echo $email_to ?>,</p>
```
Comment (LukasReschke) [inline page-tuberlin-whitepapersubmit.php]: This is insecure. Please fix this. Thanks… You just introduced a XSS to our web site which is very bad… 

Please consider switching to a saner system than plain PHP files if those cause such issues all the time.

Comment (LukasReschke) [inline page-tuberlin-whitepapersubmit.php]: Awesome we have this vulnerability all over our website! 🚀 

https://github.com/nextcloud/nextcloud.com/search?utf8=%E2%9C%93&q=%22echo+%24email_to%22&type=

Comment (MariusBluem) [inline page-tuberlin-whitepapersubmit.php]: Saw no problem, since this was done all over the website the same 😩

**Pull request merged**